plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -389,6 +390,15 @@ class ServicesTable extends React.Component {
     return <span>{Units.formatResource(prop, resource)}</span>;
   }
 
+  renderVersion(prop, service) {
+    if (!service.getVersion) {
+      return <span />;
```

Comment:
I have seen us using `null` instead if `<span />`, not a blocker for me though (as long a empty span isn't rendered)